src/createQueryGroup.ts

```diff
@@ -50,6 +49,7 @@ export function createQueryGroup<
     lastExecuted.value = Date.now()
     executing.value = false
     
+    setTags()
```

Comment:
I'd like this to be in the try so it's only called when the response changes.

Reply:
@pleek91 I pushed a change that prevents modifying tags if !executed, I think this should satisfy the requirements you're looking for. I personally am not sure the complexity is worth the theoretical gains, since setting tags seems super inexpensive. Which is good because the likelihood that execute gets called, maybe on an interval, and has the same response will definitely re-set tags. I considered even trying to account for that but I think the cost of trying to determine same response is probably 10x more expensive than just re-setting them.

Reply:
I don't think this needs to be complex at all. I see why it couldn't go in the try. But I think we can still just check executed in our tag management function. See https://github.com/kitbagjs/query/pull/17 for my take on this.